pipelinewise/fastsync/postgres_to_snowflake.py

```diff
@@ -126,8 +127,11 @@ def sync_table(table: str, args: Namespace) -> Union[bool, str]:
         # Load into Snowflake table
         snowflake.copy_to_table(s3_key_pattern, target_schema, table, size_bytes, is_temporary=True)
 
-        # Delete all file parts from s3
         for s3_key in s3_keys:
+            if archive_load_files:
+                snowflake.copy_to_archive(s3_key, tap_id, table)
+
+            # Delete all file parts from s3
```

Comment:
`archive_load_files: True`  is a standard option in snowflake target and needs to be compatible with every tap. This unfortunately means we need to  add this change to all the supported fastsync-to-snowflake files.

Can you please make the same change for the other 3 supported dbs and apply the same change that you made in in this file?

* [mysql-to-snowflake.py](https://github.com/transferwise/pipelinewise/blob/master/pipelinewise/fastsync/mysql_to_snowflake.py)
* [s3-csv-to-snowflake.py](https://github.com/transferwise/pipelinewise/blob/master/pipelinewise/fastsync/s3_csv_to_snowflake.py)
* [mongodb-to-snowflake.py](https://github.com/transferwise/pipelinewise/blob/master/pipelinewise/fastsync/mongodb_to_snowflake.py)